src/components/layout/Footer.js

```diff
@@ -251,16 +250,16 @@ export default function Footer() {
           <ThemeToggler />
         </div>
         <SubText>
-          Current Version: v0.0.1{" "}
+          {t("Current Version")}: v0.0.1{" "}
           <a
             href="https://github.com/HandshakeAlliance/HNScan"
             target="_blank"
             rel="noopener noreferrer"
           >
-            Source
+            {t("Source")}
           </a>
         </SubText>
-        <SubText>&#9400; Handshake Alliance. All Rights Reserved.</SubText>
+        <SubText>&#9400; {t("copywrite")}</SubText>
```

Comment:
tisk tisk ;)

Reply:
hahaha wait what'd I do 😭 😖

Reply:
It triggered the misspell check - `copywrite` vs `copyright`

Reply:
😭